TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -23,6 +28,9 @@ class CustomNavigationController: UINavigationController, UINavigationController
         .foregroundColor: Colors.black
     ]
 
+    private let banner = StatusBarNotificationBanner(title: Constants.Banner.noInternetConnection, style: .danger)
+    private let reachability = Reachability()
```

Comment:
Seems like we're only ever accessing this in the `viewWillAppear` function to add it as the object being passed into the notification observer and then we safely unwrap in our `reachabilityChanged` function. Could we look into moving this into the `viewWillAppear` function as a local variable instead? Not 100% about memory-related implications for this object tho

Reply:
but we're also accessing `reachability` in `viewWillDisappear` to stop notifier